CHANGELOG.md

```diff
@@ -6,6 +6,17 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 
 *Note*: We try to adhere to these practices as of version [v0.2.1].
 
+
+## Version [0.3.0] - 2024-07-16
```

Comment:
Just set this to `[1.0.0]` also then it's good to go

Reply:
> Just set this to `[1.0.0]` also then it's good to go

the version in the changelog must correspond tot he number in the project.toml?

Reply:
Yes, exactly, there are supposed to be linked to each other to make it easy for people to understand what's been changed/added/deleted in specific releases.